src/components/Divider.tsx

```diff
@@ -54,7 +54,9 @@ const Divider = ({ path }: { path: Path }) => {
 
       /** Find and set the max width of our selected elements. */
       const maxWidth = Math.max(
-        ...Array.from(elements).map(element => (element.querySelector('.thought') as HTMLElement).offsetWidth),
+        ...Array.from(elements)
+          .filter(element => treeNode.classList.contains('table-col1') === element.classList.contains('table-col1'))
+          .map(element => (element.querySelector('.thought') as HTMLElement).offsetWidth),
```

Comment:
Why wouldn't it just be `!element.classList.contains('table-col1')`? And if that's the case, why not fine-tune the original `elements` selector instead of filtering it after the fact?

I can't think of a time when we need to measure col1 width, but maybe I'm missing something.

Reply:
Doing it this way causes dividers in col1, should they exist, to ignore col2 items and vice versa. It solves the problem in both directions.

Reply:
Okay, got it. Let's add a comment since it's nonobvious.

I don't love that this adds additional direct DOM access when we already have identified that this implementation suffers from too much direct DOM access. Query selectors are not captured by the type system, and they are prone to breakage if the DOM hierarchy changes. Plus, this calculation occurs on every keystroke, and the implementation is not very performance sensitive. It would have been better to identify the thoughts to be included in React, and only perform the measurement in the DOM.

I'll accept this for the sake of moving on, but I want you to be aware going forward. Thanks.